pkg/testutil/rand/model.go

```diff
@@ -131,74 +131,75 @@ func WithMessageFormat() ModelMessageOption {
 
 // ------------------Messages------------------
 
-// modelMessages generates a random model.Messages with the given
+// modelTranslation generates a random model.Translation with the given
 // count of Messages.messages and using the provided options for each message.
-func modelMessages(msgCount uint, msgOpts ...ModelMessageOption) *model.Messages {
-	messages := &model.Messages{
+func modelTranslation(msgCount uint, msgOpts ...ModelMessageOption) *model.Translation {
+	translations := &model.Translation{
```

Comment:
Singular, please. It is a single translation, not plural (multiple).

```suggestion
	translation := &model.Translation{
```